config/locales/en.yml

```diff
@@ -115,10 +115,10 @@ en:
     application_status: Status
 
   table_headings: &TABLE_HEADINGS
-    applicant_name: Applicant
-    reference: Ref. no.
-    days_passed: Business days since applications were received
-    time_passed: Days passed
+    applicant_name: Applicants name
+    reference: Reference number
+    days_passed: Business days since application was received
+    time_passed: Business days since application was received
```

Comment:
So we want to make this so that it wraps onto the second line where we want it to. Something to pair on tomorrow I think. But it's something Tim did in this PR that Will commented on about non-breaking spaces https://github.com/ministryofjustice/laa-review-criminal-legal-aid/pull/201